clients/openfga_client.go

```diff
@@ -0,0 +1,246 @@
+package clients
```

Comment:
- Use `camelCase` for constants like `INVALID_PROMPT` instead of `UPPER_CASE`.
- Avoid using global logging; consider passing a logger as an argument instead of using the default logger, which limits flexibility.
- The `parseStyleGuideChunks` logs the error using `%w` in `Printf`, which is meant for `Errorf`. Change it to `%v` or use `Errorf`.
- In `BuildReviewPrompt`, the `fmt.Sprintf` call seems incorrect. The parameters should be properly formatted to include all three style chunks and code.
- Consider adding more detailed error handling in `NewOpenFGAClient` to differentiate between initialization failures.

Other than these minor issues, the code is generally clean and structured.